website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,56 @@
+
+Appsmith provides a way to define multiple environments for datasources. It allows you to switch between different environments while developing your app in edit mode. By switching environments, you are applying the datasource configurations defined in that environment.
+
+:::info
+The multiple environments feature is only available on [Business Edition](https://www.appsmith.com/pricing). 
+:::
+
+## Environments on Appsmith
+
+Appsmith provides two environments out of the box - **staging** and **production**. Staging environment configurations should ideally point to datasources which are used for development, QA or user acceptance testing. Production environment configurations should point to datasources which have live data that the end user see on the app.
+
+In Appsmith, environments work at the workspace level. This means that all the apps within a workspace share the same configuration values for both staging and production environments.
+
+## Configure an environment
+
+ By default, all the configuration values for a datasource are shared across staging and production environments. If you want to have different values for staging and production, you can go to the **Manage Environments** page where you can provide different values for the available configuration fields for staging and production. 
+
+<VideoEmbed host="youtube" videoId="3hoIWkvL9hA" title="Configure an environment" caption="Configure an environment | MongoDB" />  
+
+* On the Explorer tab, select the **datasource** for which you want to configure multiple environments.
+* Click the **EDIT** button on the top-right corner.
+* Click **MANAGE ENVIRONMENTS** on the top-right corner.
+* Select the environment from the **Configure Environment** dropdown list.
+* Set the `Host Address` and `port` fields under the **Endpoints** section for the chosen environment. 
+* Under **Datasource Configurations** add the values by selecting the fields you want to configure.
+* Once the fields are configured, click **BIND VALUES** button on the bottom right of the page to bind the environment variables. 
+
+You can set different values in the database configuration fields for each environment.
+
+<figure>
+  <img src="/img/me-db-1.png" style= {{width:"700px", height:"auto"}} alt="Multiple environments set for PostgreSQL datasource"/>
+  <figcaption align = "center"><i>Multiple environments set for a PostgreSQL datasource</i></figcaption>
+</figure>
+
+## Switch environments
+
+In the **edit mode**, you can toggle between staging and production environments on the bottom right corner of the screen, enabling you to run queries in the environment of your choice. 
```

Comment:
@danciaclara @harshilp24 Switching environments is also possible in view mode, if the user has permission. I think we should mention that as well.